src/main/resources/reference.conf

```diff
@@ -47,6 +47,12 @@ kamon.prometheus {
       524288,
       1048576
     ]
+
+    # Per metric overrides are possible by specifying the prometheus metric name and the histogram buckets here
+    custom {
+      // example:
+      // "akka_actor_processing_time_seconds" = [0.1, 1.0, 10.0]
```

Comment:
If I understand correctly, the matching is done with the Kamon metric name, not the sanitized-for-prometheus version of it, please change the example accordingly to avoid misleading the users.

Reply:
nope, the prometheus name is used